ibmcloudant/__init__.py

```diff
@@ -37,3 +37,5 @@
 CloudantV1.prepare_request = new_prepare_request
 
 CloudantV1.set_http_client = new_set_http_client
+
+CloudantV1.set_disable_ssl_verification = new_set_disable_ssl_verification
```

Comment:
There is a comment on the source `new_set_disable_ssl_verification` about it being unused that should be removed alongside this change.

Reply:
Done in https://github.com/IBM/cloudant-python-sdk/pull/874/commits/36c2df692269a639a0104615e59bd14268d5aafa